types/index.d.ts

```diff
@@ -63,6 +63,7 @@ export const validators: {
 export interface ValidatorOptions {
     apiDefinitionsPath: string;
     reportCoverage?: boolean;
+    printCoverageToFile?: boolean;
```

Comment:
lets change `printCoverageToFile` to `exportCoverage`. we also need to document it in the readme

Reply:
done